CHANGELOG.md

```diff
@@ -1,21 +1,43 @@
+## 2.9.0-beta.1
+
+### Features ✨
+
+* Add `globe` projection. This new projection allow the design of 3d globe and can be enabled by either passing `projection: globe` to the map constructor or by calling `map.setProjection('globe')`. All layers are supported by globe except for Custom Layers and Sky.
+* Extend atmospheric `fog` with three new style specification properties: `high-color`, `space-color` and `star-intensity` to allow the design of atmosphere around the globe. ([#11590](https://github.com/mapbox/mapbox-gl-js/pull/11590))
```

Comment:
Stars can also be used as a new feature to create night skies outside of globe, no? Maybe we should mention this here (also something to keep in mind for docs examples).